### PR TITLE
UIDATIMP-582: Extend `<SearchForm>` component with additional props

### PR DESCRIPTION
**Pull request by mariia-aloshyna**
## Purpose

1. Add `idKey` prop to the component to avoid accessibility problems with duplicated `ids` when there is more than one `<SearchForm>` on the page.
2. Add `fieldName` prop to the component to be able to set the `name` to the search field.

## Ref
[UIDATIMP-582](https://issues.folio.org/browse/UIDATIMP-582)

### CODE REVIEW

File: lib/SearchForm/SearchForm.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ export function SearchForm({
   handleChange,
   handleClear,
   handleSubmit,
+  fieldName,
+  idKey,
```
Comment (yelyzavetakhokhlova) [inline lib/SearchForm/SearchForm.js:29]: Please, follow the [codstyle](https://github.com/folio-org/ui-data-export/blob/master/CODESTYLEGUIDE.md), callbacks should be placed at the end of the destructuring list.

File: lib/SearchForm/SearchForm.js [hunk 2]
```diff
@@ -55,6 +57,7 @@ export function SearchForm({
                 onChangeIndex={handleChangeIndex}
                 onChange={handleChange}
                 onClear={handleClear}
+                name={fieldName}
```
Comment (yelyzavetakhokhlova) [inline lib/SearchForm/SearchForm.js:60]: The same, please place props in correct order.

File: lib/SearchForm/tests/SearchForm-test.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ async function setupSearchForm({
   handleClear,
   handleChange,
   handleSubmit,
+  idKey,
```
Comment (yelyzavetakhokhlova) [inline lib/SearchForm/tests/SearchForm-test.js:23]: Props placement here and on component mounting

File: lib/SearchForm/SearchForm.js (hunk 1 above)
Comment (mariia-aloshyna) [reply lib/SearchForm/SearchForm.js]: done

File: lib/SearchForm/tests/SearchForm-test.js (hunk 3 above)
Comment (mariia-aloshyna) [reply lib/SearchForm/tests/SearchForm-test.js]: done

File: lib/SearchForm/SearchForm.js (hunk 2 above)
Comment (mariia-aloshyna) [reply lib/SearchForm/SearchForm.js]: done

**Review by yelyzavetakhokhlova — APPROVED** [review]

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**